### PR TITLE
turn back on corona job

### PR DESCRIPTION
**Pull request by white238**
Turns back on the Corona source job

### CODE REVIEW

File: .gitlab/build_corona.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 .src_build_on_corona:
   stage: build
   variables:
-    ALLOC_COMMAND: "srun -p pdebug -t 25 -N 1 "
+    ALLOC_COMMAND: "srun -p pbatch -t 25 -N 1 "
```
Comment (kennyweiss) [inline .gitlab/build_corona.yml:22]: I'm confused -- is `pbatch` not available? If not, should we hold off on this PR until it is?

Comment (rhornung67) [reply .gitlab/build_corona.yml:22]: I'm confused as well. We should be running CI in pbatch.

**Review by kennyweiss — APPROVED** [review]

**Comment by white238** [comment]
This is failing due to:

```
227: terminate called after throwing an instance of 'umpire::util::Exception'
227:   what():  ! Umpire Exception [spack-src/src/umpire/op/HostReallocateOperation.cpp:40]:  transform ::realloc(current_ptr=0x7fffe9c68480, old_size=1280, new_size=80000000) failed
227:     Backtrace: 18 frames
227:     0 0x28a07e No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x28a07e]
227:     1 0x29fd90 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x29fd90]
227:     2 0x285f5e No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x285f5e]
227:     3 0x285724 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x285724]
227:     4 0x2358b2 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x2358b2]
227:     5 0x236d42 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x236d42]
227:     6 0x236464 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x236464]
227:     7 0x22f967 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x22f967]
227:     8 0x2701c6 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x2701c6]
227:     9 0x24c9d1 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x24c9d1]
227:     10 0x24ddbf No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x24ddbf]
227:     11 0x24e581 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x24e581]
227:     12 0x25bd43 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x25bd43]
227:     13 0x270ef6 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x270ef6]
227:     14 0x25b599 No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x25b599]
227:     15 0x23ea8b No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x23ea8b]
227:     16 0x7fffece35493 No dladdr: /lib64/libc.so.6(__libc_start_main+0xf3) [0x7fffece35493]
227:     17 0x22f00e No dladdr: lang_13.0.0_hip/build-corona-toss_4_x86_64_ib-clang@13.0.0_hip/tests/spin_uniform_grid_test() [0x22f00e]
```

I don't have time to look at this atm.

**Comment by rhornung67** [comment]
A few minutes ago, I learned that LC folks are patching all the gitlabs (CZ, RZ, SCF) so things may appear broken at the moment. If so, they will be back soon hopefully.... 🤞

File: .gitlab/build_corona.yml (hunk 1 above)
Comment (rhornung67) [reply .gitlab/build_corona.yml:22]: Mistake #1: assuming there is anything useful in any of Rich's messages....

**Review by bmhan12 — APPROVED** [review]

**Pull request merged**